data/remnant/remnant ships.txt

```diff
@@ -1183,33 +1188,30 @@ ship "Peregrine"
 		licenses
 			"Remnant Capital"
 		"cost" 41627000
-		"shields" 16400
-		"hull" 10600
+		"shields" 15400
+		"hull" 8600
 		"required crew" 9
 		"bunks" 25
 		"mass" 350
 		"drag" 4.3
 		"heat dissipation" 0.85
-		"fuel capacity" 1100
+		"fuel capacity" 900
 		"ramscoop" 3.25
-		"cargo space" 59
+		"cargo space" 49
 		"outfit space" 552
 		"weapon capacity" 220
 		"engine capacity" 171
-		"shield generation" 4.9
+		"shield generation" 3.9
 		"shield energy" 3.0
-		"hull repair rate" 2.5
+		"hull repair rate" 1.5
```

Comment:
These changes I agree with. They still leave the Peregrine with very good regen for its size - indeed, notably more than the Automata and Wanderer medium warships - and the additional ability to disengage that the Peregrine's extreme speed and cloaking provide give it a sufficient additional advantage if playing a hit-and-run style.

Reply:
Though, that said, the values without this change aren't an issue in themselves, they're just unreasonably high compared to other content in the game. Bringing them closer to other comparable content is still a good idea, but really I think it'd be better to see other regen improved to give other ships performance more like this, rather than the reverse.